modules/backend/controllers/Auth.php

```diff
@@ -100,12 +100,14 @@ public function signin_onSubmit()
             'password' => post('password')
         ], $remember);
 
-        try {
-            // Load version updates
-            UpdateManager::instance()->update();
-        }
-        catch (Exception $ex) {
-            Flash::error($ex->getMessage());
+        if (config('cms.runUpdateManagerAfterBackendLogin', true)) {
```

Comment:
@Samuell1 If you agree with the config name change above, you'll need to change it here too.
```suggestion
        if (config('cms.updatePluginsOnLogin', true)) {
```